internal/resolver/mtaresolver_test.go

```diff
@@ -71,8 +71,12 @@ func getExpected(expected []string) string {
 	return <-out
 }
 
-func callResolveAndValidateOutput(wd, moduleName, yamlPath string, expected []string) {
-	actualStr := callResolveAndGetOutput(wd, moduleName, yamlPath)
+func callResolveAndValidateOutput(wd, moduleName, yamlPath string, expected []string, envFileNameOps ...string) {
+	envFileName := ".env"
+	if len(envFileNameOps) > 0 {
+		envFileName = envFileNameOps[0]
+	}
```

Comment:
You already have this logic inside the `Resolve` function, there is no need to duplicate it here. If an empty parameter is sent just pass it as-is.

Reply:
Done